papers/H0_I/Analysis/CRACO/Explore_full.ipynb

```diff
@@ -69,13 +69,22 @@
        "['ll',\n",
        " 'lC',\n",
        " 'params',\n",
+       " 'pzDM',\n",
+       " 'pDM',\n",
+       " 'pDMz',\n",
+       " 'pz',\n",
        " 'lEmax',\n",
        " 'H0',\n",
        " 'alpha',\n",
        " 'gamma',\n",
        " 'sfr_n',\n",
        " 'lmean',\n",
-       " 'lsigma']"
+       " 'lsigma',\n",
```

Comment:
so if this is for the full cube, you'll want to add lls0...N0 for surveys 1,2,3,4 to the below